girder/utility/server.py

```diff
@@ -218,7 +180,7 @@ def setup(test=False, plugins=None, curConfig=None):
         girderWebroot, str(routeTable[constants.GIRDER_ROUTE_ID]), appconf)
 
     # Mount static files
-    cherrypy.tree.mount(None, routeTable[constants.GIRDER_STATIC_ROUTE_ID],
+    cherrypy.tree.mount(None, '/static',
                         {'/':
                          {'tools.staticdir.on': True,
                           'tools.staticdir.dir': os.path.join(constants.STATIC_ROOT_DIR),
```

Comment:
Below on line 235, could the conditional be simplified? Is there anything in `routeTable` that's not in `pluginWebroots` other than `constants.GIRDER_ROUTE_ID`?